spec/mailers/user_mailer_spec.rb

```diff
@@ -1200,4 +1200,54 @@
       it_behaves_like "an email with a deleted work with draft chapters attached"
     end
   end
+
+  describe "obeys the set locale preference feature flag" do
```

Comment:
Is it worth adding similar tests to
* https://github.com/otwcode/otwarchive/blob/b612112bb1f8b7d846951367feda8165e516ac00/spec/mailers/comment_mailer_spec.rb
* https://github.com/otwcode/otwarchive/blob/3359d060a00ef71be7428e8755b6a036cbd13941/spec/mailers/kudo_mailer_spec.rb

as well?

Reply:
I don't really think it's worth it, the point of the test was mostly to test that the function itself works. The location of the test itself is irrelevant. You can see that I moved it around in one of the commits. Motivation there was that it would be testing a different mailer than the instructions for the manual testing on Jira.

However, I started looking into whether I could convert the test into a shared example that could be included in all mailer tests, similar to "a multipart email" etc. But I got stalled out on this and won't have time to look at it again until the weekend. Hence this progress report, because information is good.

Reply:
I can't get the shared example to work, something about the interaction between the email as a subject and the expect(...) just doesn't work.

So I'd leave the tests as is, if that's okay.

Reply:
That's fine in my books, thanks for taking a look!